extension/view/js/translation-notification-fxtranslations.js

```diff
@@ -25,13 +25,14 @@ window.MozTranslationNotification = class extends MozElements.Notification {
             <checkbox anonid="qualityestimations-check" label="" style="padding-left:5px" oncommand="this.closest('notification').onQeClick();"/>
           </hbox>
           <vbox class="translating-box" pack="center">
-            <hbox>
+            <hbox class="translate-offer-box" align="center">
               <label value="&translation.translatingContent.label;" style="display:none"/>
-              <label anonid="progress-label" value="" style="padding-left:5px"/>     
+              <label anonid="progress-label" value="" style="padding-left:5px;"/>
+              <button class="notification-button primary" label="" anonid="translateAsBrowse" oncommand="this.closest('notification').translateAsBrowse();"/>
```

Comment:
"primary" colour feels like too much when translation is enabled, very distracting. Maybe let's use the grey one like for options and survey?
<img width="1274" alt="Screen Shot 2022-04-07 at 11 26 04 AM" src="https://user-images.githubusercontent.com/2486505/162272145-5c697b2c-c4df-4f86-b706-01f3bd951f28.png">